miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -507,8 +499,18 @@ private void applyChanges(Sample target, Sample source) throws IOException {
       if (isTissueSample(target)) {
         sampleTissueService.applyChanges((SampleTissue) target, (SampleTissue) source);
       }
-      if (isAnalyteSample(target)) {
-        sampleAnalyteService.applyChanges((SampleAnalyte) target, (SampleAnalyte) source);
+      if (isAliquotSample(target)) {
+        SampleAliquot saTarget = (SampleAliquot) target;
+        SampleAliquot saSource = (SampleAliquot) source;
+        saTarget.setGroupId(saSource.getGroupId());
```

Comment:
groupId and groupDescription are members of SampleAdditionalInfo, and are already copied over in sampleAdditionalInfoService.applyChanges (called above)